components/card/stories/card.mdx

```diff
@@ -0,0 +1,62 @@
+import { ArgTypes, Canvas, Meta, Description, Title } from '@storybook/blocks';
+
+import * as CardStories from './card.stories';
+
+<Meta of={CardStories} title="Docs" />
+
+<Title of={CardStories} />
+<Description of={CardStories} />
+
+## Standard
+
+<Canvas of={CardStories.Default} />
+
+### No image
+
+<Canvas of={CardStories.NoImage} />
+
+### Focused
+
+<Canvas of={CardStories.Focused} />
+
+### Selected
+
+<Canvas of={CardStories.Selected} />
+
+## Quiet
+
+<Canvas of={CardStories.Quiet} />
+
+### File
+
+<Canvas of={CardStories.QuietFile} />
+
+### Folder
+
+<Canvas of={CardStories.QuietFolder} />
+
+### Focused
+
+<Canvas of={CardStories.QuietFocused} />
+
+### Selected
+
+<Canvas of={CardStories.QuietSelected} />
+
+## Horizontal
+
+<Canvas of={CardStories.Horizontal} />
+
+## Asset preview
+A standard card with a full-sized asset preview.
+
+<Canvas of={CardStories.AssetPreview} />
+
+## Gallery
+A gallery card for an image.
+
+<Canvas of={CardStories.Gallery} />
```

Comment:
Is there a reason this one looks different from the Docs site with the placement of the "jpg" and missing date? (Doesn't seem like a problem for this PR though.)
![image](https://github.com/adobe/spectrum-css/assets/54716846/18ef9012-7756-41ab-8f7f-5fd961524f6e)

Reply:
Great question. It looks like this is because the "Subtitle" option was deprecated ([migration guide note](https://pr-2783--spectrum-css.netlify.app/card#migrationguide)), and that's what the gallery example was showing before.